modules/history/src/views/full/index.jsx

```diff
@@ -48,23 +49,26 @@ export default class FullView extends React.Component {
     return sessionId !== this.state.currentConversation || currentConversation.count !== receivedData.messageCount
   }
 
-  getMessagesOfConversation = async sessionId => {
-    const { data } = await this.props.bp.axios.get(`/mod/history/messages/${sessionId}`)
+  getMessagesOfConversation = async (sessionId, filters) => {
+    let ressourceUrl = `${BASE_API_PATH}/messages/${sessionId}`
+    ressourceUrl += filters && filters.flag ? '?flag=true' : ''
+    const { data } = await this.props.bp.axios.get(ressourceUrl)
```

Comment:
When adding query parameters, check add them as params without editing the url: axios.get(ressourceUrl, { params: {} })